Makefile

```diff
@@ -0,0 +1,57 @@
+NAMESPACE=app-metrics
```

Comment:
We've been using the syntax mobile-<service-name> for other places, should we do that here also?

Reply:
We can definitely change it. What other places is it like that? The UPS one just says unifiedpush and, well, the mobile-developer-console and mobile-security-service already had "mobile-" in their own names :P

Reply:
This is true. As I mentioned in my previous comment there's a couple of different naming options. Perhaps we should ping Paul/Chris/Wei in the trello card to get an agreement on that rather than discussing it here?

Reply:
Feel free to :+1: Note that this is only a default for the Makefile though, so it should only really be needed for local development. People can really install into whatever namespace they want. I think the opinions of the people you mentioned would matter more for product-level namespace that we use (which, yeah, we should get their guidance on for consistency :smiley: )